flocker/control/test/test_persistence.py

```diff
@@ -123,6 +134,24 @@ def saved_again(_):
         d.addCallback(saved_again)
         return d
 
+    @validate_logging(
+        lambda test, logger:
+        test.assertEqual(len(logger.flush_tracebacks(ZeroDivisionError)), 1))
```

Comment:
if this fails because, say, two tracebacks are logged ... is the test failure meaningful (I'm guessing not... because `2 != 1`?)